pkgs/by-name/el/element-web-unwrapped/package.nix

```diff
@@ -50,6 +57,12 @@ stdenv.mkDerivation (
       runHook preBuild
 
       export VERSION=${finalAttrs.version}
+
+      pushd packages/shared-components
+      yarnOfflineCache=${offlineCacheSharedComponents} yarnConfigHook
```

Comment:
Could you maybe use `finalAttrs.offlineCacheSharedComponents` so that this can be overriden?